### PR TITLE
Using standard anomaly params for scalar input

### PR DESCRIPTION
**Pull request by rhyolight**
This PR replaces the model params you are using with those we use at Numenta for all anomaly detection applications. We have found that there are a few sets of params that work very well for anomaly detection on all scalar input. This means that your predictions are not going to be as good, but since the main function of this app is anomaly detection, perhaps that is okay.

### CODE REVIEW

File: engine/index.py [hunk 1]
```diff
@@ -217,6 +214,9 @@ def run(self):
                 val = self.queue_in.get(True, 1)
             except Empty:
                 continue
+            # Not sure why I needed to check this:
+            if val[self.sensor] is None:
+                continue
```
Comment (rhyolight) [inline engine/index.py:219]: @JonnoFTW Kept failing without this. Probably just a data issue. Feel free to remove, but I could not run without it.

**Comment by JonnoFTW** [comment]
Thanks for that!

**Comment by rhyolight** [comment]
I hope it works better! I've been running the program for 20 minutes now, and I don't think it's going to end. I'll probably kill it.

**Comment by JonnoFTW** [comment]
It should give you a progress indicator with a time estimate. 20 mins is
usually how long it takes for me to run the intersection I gave you on an 8
core machine.

On 28 October 2015 at 09:31, Matthew Taylor notifications@github.com
wrote:

> I hope it works better! I've been running the program for 20 minutes now,
> and I don't think it's going to end. I'll probably kill it.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/JonnoFTW/htm-models-adelaide/pull/3#issuecomment-151669857
> .

## 

_Jonathan Mackenzie_
BEng (Software) Hons
PhD Candidate, Flinders University

**Comment by rhyolight** [comment]
Yeah I got a progress indicator, but it did not progress very quickly.
Also, when I looked at my CPU usage, only 4 cores were being used, and they
were quite underutilized.

let me know how this works for you today.

---

Matt Taylor
OS Community Flag-Bearer
Numenta

**Pull request merged**